### PR TITLE
feat(docs): Added Documentation for Stackblitz Setup

### LINKED ISSUE #258: Document setup using stackblitz
Stack blits can be used to develop cv-frontend without having to setup everything locally.

https://stackblitz.com/~/github.com/CircuitVerse/cv-frontend-vue

### PR DESCRIPTION
**Pull request by niladrix719**
Fixes #258 

#### Describe the changes you have made in this PR -

Added Documentation for Stackblitz Setup

### Screenshots of the changes (If any) -
Screenshot 2024-01-16 at 8 50 16 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,18 @@ npm install
 npm run dev
 ```
 
+## Setting up on cloud with Stackblitz Codeflow
+[Codeflow](https://developer.stackblitz.com/codeflow/what-is-codeflow) is a one-click integration with GitHub for seamless coding workflows.
+
+1. Initiate the setup process by clicking on the following button:
+
+[![Open in Codeflow](https://developer.stackblitz.com/img/open_in_codeflow.svg)](https:///pr.new/CircuitVerse/cv-frontend-vue)
```
Comment (Arnabdaz) [inline README.md:28]: pr.new is used for pr reviews follow 
[https://developer.stackblitz.com/guides/integration/open-from-github](https://developer.stackblitz.com/guides/integration/open-from-github)
to setup the documentation

**Review by Arnabdaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (niladrix719) [reply README.md]: thank you for the correction

File: README.md [hunk 2]
```diff
@@ -20,6 +20,18 @@ npm install
...
+## Setting up on cloud with Stackblitz
+[StackBlitz](https://developer.stackblitz.com/guides/user-guide/what-is-stackblitz) is an instant fullstack web IDE for the JavaScript ecosystem.
+
+1. Initiate the setup process by clicking on the following button:
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/~/github.com/CircuitVerse/cv-frontend-vue)
+
+2. Once the setup is complete, a Preview URL will be displayed in the browser window. Append /simulatorvue/ to your URL to access the simulator.
+```
+https://<your_url>/simulatorvue/
```
Comment (Arnabdaz) [inline README.md:32]: instead of `your_url` write `preview_url`

**Review by Arnabdaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Arnabdaz) [inline README.md:32]:
```suggestion
https://<preview_url>/simulatorvue/
```

**Review by Arnabdaz — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -20,6 +20,18 @@ npm install
...
 ```
 
+## Setting up on cloud with Stackblitz
+[StackBlitz](https://developer.stackblitz.com/guides/user-guide/what-is-stackblitz) is an instant fullstack web IDE for the JavaScript ecosystem.
+
+1. Initiate the setup process by clicking on the following button:
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/~/github.com/CircuitVerse/cv-frontend-vue)
+
+2. Once the setup is complete, a Preview URL will be displayed in the browser window. Append /simulatorvue/ to your URL to access the simulator.
```
Comment (Arnabdaz) [inline README.md:30]:
```suggestion
2. Once the setup is complete, a Preview URL will be displayed in the browser window. Append `/simulatorvue/` to your URL to access the simulator.
```

**Review by Arnabdaz — CHANGES_REQUESTED** [review]

**Review by Arnabdaz — APPROVED** [review]
LGTM :+1:

**Pull request merged**